easybuild/easyconfigs/r/RepeatMasker/RepeatMasker-4.1.2-p1-foss-2020b.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'Tarball'
+
+name = 'RepeatMasker'
+version = '4.1.2-p1'
+
+homepage = 'https://www.repeatmasker.org/'
+description = """RepeatMasker is a program that screens DNA sequences for interspersed repeats
+ and low complexity DNA sequences."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

Comment:
@jfgrimm is there a specific reason why you've moved it to `foss` rather than `gompi`?

Reply:
Dependency on h5py, which is in foss

Reply:
Yes, new in this release - thanks for clarifying.